charts/localstack/templates/deployment.yaml

```diff
@@ -50,6 +50,11 @@ spec:
               port: {{ .Values.service.edgeService.name }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
```

Comment:
Should we also wrap this entire block (lines 53-57) in a `if .Values.mountDind.enabled` block?

Or, alternatively:
```
           volumeMounts:
            {{- if and .Values.mountDind.enabled .Values.mountDind.forceTLS }}
            - name: dind-tls
              mountPath: /opt/docker/tls
            {{- end }}
```

Reply:
heh sure, changing ^^ i left it this way cause i am about to post another PR for supporting persistency :smile: